pkg/commands/pfadd.ts

```diff
@@ -0,0 +1,13 @@
+import { Command, CommandOptions } from "./command.ts";
+
+/**
+ * @see https://redis.io/commands/pfadd
+ */
+export class PfAddCommand<TData = string> extends Command<number, number> {
+  constructor(
+    cmd: [key: string, ...members: TData[]],
```

Comment:
Maybe we should allow passing array like `[value1, value2, value3]`. Similar to geo_hash

``` 
cmd: [string, ...(TData[] | TData[])],
```